pkg/controller/generic_reconciler.go

```diff
@@ -93,27 +96,37 @@ func (gr *GenericReconciler) AddToManager(mgr manager.Manager) error {
 // Start validating the given object kind every interval.
 func (gr *GenericReconciler) Start(ctx context.Context) error {
 	var log = logf.Log.WithName("controller")
+	ticker := time.Tick(60 * time.Second)
+	defer gr.workqueue.ShutDown()
+
+	workerCount := 3
```

Comment:
Running multiple workers only makes sense, if you increase QpS settings, otherwise having multiple workers waiting for QpS is not helping throughput :)

Reply:
i have added environment variables to configure:
- qps 
- enqueue interval
- number of workers

Therefore these values could be tunned during deployment.